src/main/studio/index.js

```diff
@@ -63,8 +67,17 @@ class SpringRollStudio {
    * Handler for EVENTS.CREATE_PROJECT_TEMPLATE event.
    * @memberof SpringRollStudio
    */
-  createProjectTemplate() {
-    console.log('[createProjectTemplate] Missing implementation');
+  createProjectTemplate(event, data) {
```

Comment:
This could be async as well, which might clean it up some.

Reply:
Fixed